app/components/c-product-detail.vue

```diff
@@ -38,6 +38,23 @@
             <e-heading underline tag-name="h2" color="gray">{{ $t('c-product-detail.productDescriptionTitle') }}</e-heading>
             <div :class="b('description-text')" v-html="product.description"></div>
           </div>
+          <c-collapse-group>
+            <c-collapse v-if="product.media_attributes"
```

Comment:
- [ ] v-if doesn't work if its an empty array
- [ ] Use product.media_attributes.productDataSheet. I sent you an example object in slack